### DISCUSSION TITLE
mjml2html dissapear display:none

### DISCUSSION THREAD

**Discussion by JoanCodinachUoc** [GrapesJS/grapesjs#5469] [Show and tell]
When with the layers i set anyone invisible, in the mjml code puts a display:none and the layer dissapear, but when we use mjml2html and we get the html code this display:none dissapears and the layer shows again. We want that the html code keep this display:none in order to not show when we send the email. I don't know if it's a bug or mjml2html don't support display:none but someone has any idea ? 

Thank you

**Comment by pepoteloko** [comment]
You could try to set size to 0px to hide the section on the email.

**Reply by JoanCodinachUoc** [reply]
You mean that subscribe to component:update and when the atrtibute update visible = false, modify the height and width to 0px in order to hide the component ?  Could be a solution but i know if it's the best one.

But Why mjml2html don't inherit display:flex ? It's done on porpuse ?

thanks regardless

**Reply by pepoteloko** [reply]
No, more easy :)
Instead of make disappear some div with display:none, make it disappear with size 0, with:0px, height:0px.
We add a tracking pixel in some emails, using an image of size width=1 height=1
